packages/dd-trace/src/appsec/telemetry/common.js

```diff
@@ -3,11 +3,15 @@
 const DD_TELEMETRY_REQUEST_METRICS = Symbol('_dd.appsec.telemetry.request.metrics')
 
 const tags = {
+  BLOCK_FAILURE: 'block_failure',
+  EVENT_RULES_VERSION: 'event_rules_version',
+  INPUT_TRUNCATED: 'input_truncated',
   REQUEST_BLOCKED: 'request_blocked',
+  RATE_LIMITED: 'rate_limited',
```

Comment:
Seems that you are ordering alphabetically, so RATE is before REQUEST.

```suggestion
  RATE_LIMITED: 'rate_limited',
  REQUEST_BLOCKED: 'request_blocked',
```